language-server/modules/langserver-core/src/test/resources/codeaction/create-function/config/createUndefinedFunctionInCompoundStatement4.json

```diff
@@ -0,0 +1,14 @@
+{
+  "range": {
+    "start": {
+      "line": 6,
+      "character": 9
+    },
+    "end": {
+      "line": 6,
+      "character": 17
+    }
+  },
+  "source": "testproject/modules/module2/module2.bal",
+  "expected": []
```

Comment:
We should suggest the code action for this scenario.

Reply:
Remove this test